packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -148,7 +148,7 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
           <EuiSplitPanel.Inner
             grow={false}
             color={banner.backgroundColor}
-            paddingSize="none"
+            paddingSize="xs"
```

Comment:
what do you think about also changing the one line 158 to `xs` as well? Right now we're loosing a lot of vertical space in preview mode with the banner plus the cross icon. `xs` still looks good while gaining a few pixels...

Reply:
in rule preview, top margin was set to `-15px`, i updated alert preview to do the same, that should help gain more real estate